index.js

```
@@ -0,0 +1,46 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const path = require("path");
+const fse = require("fs-extra");
+const contentRouter = require("./contentRouter");
+const contentTypesRouter = require("./contentTypesRouter");
+
+require("dotenv").config();
+async function server(root, gitRepositoryRoot) {
```

How about passing an object with properties `publicBaseUrl`, `dbDir`? `gitRepositoryRoot` cam be dropped as `dbDir` must be under git repo and we can traverse parents to find git root (I found lib [find-git-root](https://www.npmjs.com/package/find-git-root) which should do the trick)

Also, we should probably validate inputs and throw an exception if folder doesn't exist or not provided.